fahrschein-http-spring/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'fahrschein.java-conventions'
+    id 'fahrschein.maven-publishing-conventions'
+}
+
+dependencies {
+    api project(':fahrschein-http-api')
+    api "org.springframework:spring-web:${property('spring.version')}"
+    testImplementation "com.squareup.okhttp3:okhttp:${property('okhttp.version')}"
+    testImplementation(testFixtures(project(':fahrschein-http-test-support')))
+}
+
+publishing.publications.maven.pom.description = 'Fahrschein HTTP Spring Adapter'
```

Comment:
What about using a common description for the HTTP client projects. So maybe going with `Fahrschein HTTP Client using Spring Adapter`.

Reply:
I copied the descriptions from the pom.xml, having a common format is reasonable. 👍